src/resolvers/projectResolver.test.ts

```diff
@@ -3641,6 +3641,22 @@ function similarProjectsBySlugTestCases() {
     );
     assert.isUndefined(currentViewedProject);
   });
+  it('should not throw error if project doesnt have any category ', async () => {
+    const viewedProject = await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      slug: String(new Date().getTime()),
+      categories: [],
+    });
+
+    const result = await axios.post(graphqlUrl, {
+      query: fetchSimilarProjectsBySlugQuery,
+      variables: {
+        slug: viewedProject.slug,
+      },
+    });
+    assert.isArray(result.data.data.similarProjectsBySlug.projects);
```

Comment:
You know I dont care about the result item here, we just need to get array, because before fixing the issue it was returning `null` for data and was throwing exception, so I just need to make sure it doesn't throw exception anymore and the result number is not important in this test case